dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -2463,6 +2465,14 @@ void DeltaMergeStore::applyAlters(
             // primary key flag
             original_table_handle_define.name = pk_names[0];
         }
+        if (table_info.value().get().replica_info.count == 0)
+        {
+            LOG_FMT_INFO(
+                log,
+                "TiFlash replica is 0. Table id:{}",
+                table_info.value().get().id);
```

Comment:
I think you can print the replica num in `SchemaBuilder:: applySetTiFlashReplicaOnPhysicalTable` instead of adding another log

https://github.com/pingcap/tiflash/blob/7ab8142917deb0d2c7670d867e2c7f671d00e7eb/dbms/src/TiDB/Schema/SchemaBuilder.cpp#L1257-L1264